### ISSUE TITLE
Accessibility: Download link with same name creates an accessibility issue if multiple pdf blocks are used in the same page

### ISSUE THREAD

**Issue by rnunez80** [eea/volto-pdf-block#21]
Link Text is the label for a link. The Link Text is also the content read by a screen reader for a link. All web visitors should be able to know where a link goes based solely on the Link Text without knowing the surrounding context of the link. 

Impact: minor

Violations
WCAG 2.1 Level AAA: [2.4.9 Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only)

Element Location
.pdf-viewer-block:nth-child(3) > .mgrpdf__wrapper.pdf-wrapper > .pdf-toolbar.pdf-toolbar-top > div:nth-child(2) > a

Element Source
<a href="https://yoursite/your.pdf/@@download/file">

Failure Summary
The link text utilized for this link has been utilized for another link within the webpage that goes to a different URL. Users can update the Link Text within the link to be more specific. ARIA attributes, aria-label or aria-labelledby, can also be used to provide additional Link Text details.

Steps to recreate:
create a new page add multiple PDF blocks going to different files
run accessibility test (Dubbot used in this case)

Also: the buttons are reported as empty links by WAVE accessibility tool
